lms/static/js/student_account/components/PasswordResetConfirmation.jsx

```diff
@@ -0,0 +1,140 @@
+/* globals gettext */
+
+import 'whatwg-fetch';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { Button, StatusAlert } from '@edx/paragon/static';
+
+import PasswordResetInput from './PasswordResetInput';
+
+// NOTE: Use static paragon with this because some internal classes (StatusAlert at least)
+// conflict with some standard LMS ones ('alert' at least). This means that you need to do
+// something like the following on any templates that use this class:
+//
+// <link type='text/css' rel='stylesheet' href='${STATIC_URL}paragon/static/paragon.min.css'>
+//
+
+class PasswordResetConfirmation extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      password1: '',
```

Comment:
Naming nit: it might be clearer that `password1` is actually the password value and `password2` is the password confirmation value. Not that there are two passwords that are used to authenticate a user, for example.